src/pages/newsletter.astro

```diff
@@ -0,0 +1,31 @@
+---
+import Layout from "../layouts/Layout.astro"
+import Heading from "../components/Heading.astro"
+import CtaButton from "../components/CtaButton.astro"
+// import { Img } from "astro-imagetools/components"
+
+const title = `Newsletter`
+
+// To update with new form contents:
+// * replace the contents starting with <!-- Note : with the new form contents
+// Remove the two META lines at the top of the form
+// Change the <script> tag to <script is:inline>
+// Note that the URL in the form generator in Zoho must match the actual URL here, or else use "*" there
+---
+
+<Layout
+    title={title}
+    description={`Sign up for our newsletter.`}
+>
+  <main class="max-w-4xl mx-auto mb-24">
+    <Heading title={`Sign up for our newsletter`} />
+
+    This is a low-volume newsletter, usually just once a month. We'd love to be able to keep in touch with you.
```

Comment:
I think it’s nice to quantify how frequently it goes out. Might be good to say what it’s about so I know whether I want to subscribe. (Is it enough to keep up with Discord announcements, or follow blog posts, for example, or is there something new or unique about the newsletter?)

Reply:
More important is to figure out how to allow unsubscribe, which currently is "reply to randy".